app/controllers/concerns/cloud/stripeable.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Cloud
+  module Stripeable
+    extend ActiveSupport::Concern
+
+    private
+
+    def stripe_product
+      @stripe_product ||= stripe_subscription.items&.data&.first&.price&.product
+    end
+
+    def stripe_subscription
+      @stripe_subscription ||= stripe_customer.subscriptions&.data&.first
+    end
+
+    def stripe_customer
+      return unless mapzy_cloud?
```

Comment:
```
    def stripe_customer
      return unless mapzy_cloud?
      ...
     end
```

as `stripe_customer` can be nil, we should be more careful in the other methods calling it (`stripe_subscription` and `stripe_product`).

Reply:
True, but in the current case the other methods never call `stripe_customer` if it's `blank`. I guess should be good for now?